README.md

```diff
@@ -22,7 +22,10 @@ sudo docker build . -t sydneyinformaticshub/rnaseq-rstudio:4.1.0
 
 ## Run with Docker
 
-To run this container, mount a directory housing your data (specify which paths you need to mount), and run the RStudio server instance with: 
+To run this container:
+
+- Start docker
+- Mount a directory housing your data (specify which paths you need to mount), and run the RStudio server instance using the following command: 
```

Comment:
Could you add a quick example of this here? It might make the user experience a bit more streamlined if they don't know how to do this.

Reply:
Fixed in other branch